src/GenAPI/Microsoft.DotNet.GenAPI.Task/Microsoft.DotNet.GenAPI.Task.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(TargetFrameworkForNETSDK)</TargetFrameworks>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <PackageType>MSBuildSdk</PackageType>
```

Comment:
We don't want this package to be an msbuild sdk package. `Microsoft.NET.Sdk` or `Microsoft.Build.NoTargets` are examples of msbuild sdks. This package is just a tool that will be consumed  via msbuild props and targets files but not as an sdk.